frontend/src/app/new-claim/new-claim.component.ts

```diff
@@ -52,18 +57,27 @@ export class NewClaimComponent {
   }
 
   areFieldsUntouched(): boolean {
-    return this.vin.untouched || this.color.untouched || this.model.untouched;
+    return !(this.vin.value && this.model.value && this.year.value);
```

Comment:
If prettier did this, color me impressed.

Reply:
If you're calling me pretty, thanks, because I did it

Reply:
Oh, then that's stupid. You IDIOT!